examples/complete/main.tf

```diff
@@ -2,11 +2,38 @@ provider "aws" {
   region = var.region
 }
 
+data "aws_caller_identity" "current" {}
+
+locals {
+  sqs_queue_arn_prefix = "arn:aws:sqs:${var.region}:${data.aws_caller_identity.current.account_id}"
+}
+
+resource "aws_sqs_queue" "this" {
+  count      = 2
```

Comment:
Let's use a `for_each` here and then below we can use `aws_sqs_queue.default[xyz].arn` instead of hard coding the string value

Reply:
updated